_projects/2022/100386498/100386498.Rmd

```diff
@@ -0,0 +1,390 @@
+
+Moreover, I have decided to keep the logarithmic transformation because it helps the visualization of the data. It also helps to the representation of the information that the axes are not shown in logarithmic scale, and are rather displayed in the original scale.
+
+As in the original graph it was a bit confusing to know precisely which dot the label was referring to, by now having four facets, there is more space to specify each country. To do this I used `library(ggrepel)`.
+
+In addition, to enhance the colors of the dots I used the color scheme by [Carto](https://carto.com/carto-colors/) to make more visible the different groups.
+
+```{r fig.show='hide'}
+p1 <- ggplot(qntile, aes(HFCE, national_povt_line, color=country_class)) + 
+  geom_point(data = transform(qntile,
+     country_class = NULL), colour = "grey80") + 
+  geom_point(alpha=0.7) +
+  ggrepel::geom_text_repel(aes(label=country)) +
+  scale_color_manual(values = c("#5F4690","#38A6A5", "#73AF48", "#E17C05")) +
+  facet_grid(~country_class, scales = "free") + 
+  theme_bw()
+```
+
+```{r echo=FALSE, fig.width=15,fig.height=12,fig.align = "center"}
+p1 <- p1 + labs(title = "<span style='color: #CC503E;'>1c<span><span style='color: #000000;'> Richer
+          countries tend to have higher national poverty lines<span>",
+          subtitle="Class country",
+       x = "Household final consumption expenditure per capita per day, 2000-12 (2011 PPP $)",
+       y = "National per capita per day poverty line, 2000–12 (2011 PPP $)",
+       caption = "Source: Jolliffe, D. M., and E. B. Prydz. 2016, Estimating International Poverty Lines from Comparable National Thresholds, Policy Research Working Paper WPS 7606, Washington, DC: World Bank Group.") +
+  scale_x_log10()+
+  scale_y_log10()+
+  theme(text = element_text(size = 16),
+        legend.position="top",
+        legend.title = element_blank(),
+        legend.background = element_rect(size=0.5, linetype="solid", 
+                                  colour ="#b2b2b2"),
+        plot.title = element_markdown (face = "bold", hjust = 0),
+        plot.subtitle = element_markdown(face = "bold",hjust = 0.5),
+        axis.title.x  = element_markdown(),
+        axis.title.y = element_markdown(),
+        plot.caption.position = "plot",
+        plot.caption = element_text(hjust = 0, vjust = 0.5)) +
+  guides(color = guide_legend(override.aes = list(size = 6)))
+p1
+```
+
+### Interactive chart
+Another alternative visualization is to plot an interactive scatterplot by using `ploytly`. This way more information is added to every dot without increasing data density:
+
+  - Country
+  - Year
+  - National poverty line
+  - Household final consumption
+  - GNI
+
+```{r fig.width=100,fig.height=90,fig.align = "center"}
```

Comment:
Currently, this plot doesn't fit into my screen vertically. A `fig.asp=0.7`  (after removing the current `fig.height`) may look better.